bootloader/grub.go

```diff
@@ -481,6 +490,9 @@ func (g *grub) getGrubRecoveryModeTrustedAssets() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	if osutil.FileExists(filepath.Join(g.rootdir, assets.fallbackBinary)) {
```

Comment:
I need to think about this,  so far the list was a static property, now it's based on non-encrypted partition content.

Reply:
we can look this up from the gadget.yaml too, as it must declare/install files into those file targets. And gadget.yaml should be verified and accessible when this is running.